{'project_id': '3130', 'year': '1984', 'x': -1824120, 'y': 1416000, 'plot_id': '3226', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='3226', x=-1824120, y=1416000)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='3226', x=-1824120, y=1416000)
3226
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_3226_3130.p'.
Failed qa: 1984-03-23T17:50:01.848018Z
.......Failed qa: 1984-04-27T17:55:40.421040Z
..............Failed qa: 1984-05-13T17:55:53.4110130Z
..............Failed qa: 1984-05-29T17:56:31.373047Z
..............Failed qa: 1984-06-14T17:56:49.585056Z
..............Failed qa: 1984-06-30T17:57:02.070054Z
..............Failed qa: 1984-07-16T17:57:29.0630310Z
Failed qa: 1984-08-26T17:52:22.711056Z
.......Failed qa: 1984-10-04T17:58:26.5190880Z
..............Failed qa: 1984-11-21T17:58:24.660541Z
..............Data has been pickled and saved to '/efs/timesync/1984/audit/1984_3226_3130_qa_rejects.p'.
Success tracking goes here
